interfaces/default/js/kodi.js

```diff
@@ -217,6 +217,13 @@ function loadMovies(options) {
 
                     movieAnchor.append($('<h6>').addClass('title').html(shortenText(movie.title, 12)));
 
+                    if (kodi_vod_enabled){
+                        movieItem.append($('<h6>').addClass('title').append($('<a>').attr('href', '#').html("Play in Browser").click(function(e) {
```

Comment:
I would like to remove this and rather have a button in the modal. What do you think about that?